src/path.ts

```diff
@@ -0,0 +1,157 @@
+// 'path' module extracted from Node.js v8.11.1 (only the posix part)
+// transplited with Babel, some manual changes
```

Comment:
__thought:__ would be nice to have a link to the source in this comment

Reply:
I copied this note from the polyfill lib. Not sure where the OG source code is 😅